src/HealthcareApis/HealthcareApis/help/Remove-AzHealthcareApisService.md

```diff
@@ -0,0 +1,201 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.HealthcareApis.dll-Help.xml
+Module Name: Az.HealthcareApis
+online version: https://docs.microsoft.com/en-us/powershell/module/az.healthcareapisservice/remove-azhealthcareapisservice
+schema: 2.0.0
+---
+
+# Remove-AzHealthcareApisService
+
+## SYNOPSIS
+Deletes a service instance.
+
+## SYNTAX
+
+### ServiceNameParameterSet (Default)
+```
+Remove-AzHealthcareApisService -Name <String> -ResourceGroupName <String> [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### InputObjectParameterSet
+```
+Remove-AzHealthcareApisService [-InputObject <PSHealthcareApisService>] [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceIdParameterSet
+```
+Remove-AzHealthcareApisService [-ResourceId] <String> [-AsJob] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Deletes a service instance.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Remove-AzHealthcareApisFhirService -Name MyService -ResourceGroupName MyResourceGroup
+
+ResourceGroupName Name 
+----------------- ----------- 
+MyResourceGroup   MyService
+```
+
+Deletes the existing HealthcareApis service with the provided name within a provided resource group.
+
+### Example 2
+```powershell
+PS C:\> Remove-AzHealthcareApisFhirService -ResourceId myResourceId
+```
+
+Deletes the existing HealthcareApis service with the provided ResourceId.
+
+### Example 3
+```powershell
+PS C:\> Remove-AzHealthcareApisFhirService -InputObject PSHealthcareApisService
```

Comment:
@poadhika same comment here -- `PSHealthcareApisService` is interpreted as a string, not an object, so you would need to do the following:

```powershell
$InputObject = Get-AzHealthcareApisService -ResourceGroupName MyResourceGroup -Name MyService
Remove-AzHealthcareApisService -InputObject $InputObject
```

or, better yet, you can display the piping scenario:

```powershell
Get-AzHealthcareApisService -ResourceGroupName MyResourceGroup -Name MyService | Remove-AzHealthcareApisService
```